src/tools/perf/perftest.c

```diff
@@ -372,6 +372,9 @@ static void usage(struct perftest_context *ctx, const char *program)
     printf("     -B             Register memory with NONBLOCK flag.\n");
     printf("     -C             Use wildcard for tag tests.\n");
     printf("     -S             Use synchronous mode for tag sends.\n");
+    printf("     -r <mode>      Stream receive mode. (recv_data)\n");
```

Comment:
i think recv should be the default since it symmetric for 'stream_send_nb', the 'recv_data' mode is an optimization
also can say in the help: "Receive mode for stream tests. (recv)\n"